lib/stencil-bundle.js

```diff
@@ -354,14 +357,16 @@ function bundleThemeFiles(archive, themePath, configuration) {
  * Archive all generated files (ex. parsed files)
  * @param {Archive} archive
  * @param {Object} taskResults
+ * @returns {Array} 
  */
 function bundleParsedFiles(archive, taskResults) {
     const archiveJsonFile = (data, name) => {
         archive.append(JSON.stringify(data), { name });
     }
-
+    const templatesArr=[];
```

Comment:
should this be a `var` since it can be modified in this method?

Reply:
Unrelated to this: if the idea is that we might (at a later date) have a series of errors to pass back, maybe this should be called `errorsArr` or something along those lines?

Reply:
No to first as const creates a read only reference to a value.  We aren't changing what it refers to.  The second one may be a good idea.

Reply:
@megdesko You mean `let` ;)